Makefile

```diff
@@ -220,6 +219,7 @@ $(_TEMPDIR)/%_podman.tar: podman/Containerfile podman/setup.sh $(wildcard base_i
 	podman build -t $*_podman:$(call err_if_empty,IMG_SFX) \
 		--build-arg=BASE_NAME=$(subst prior-,,$*) \
 		--build-arg=BASE_TAG=$(call err_if_empty,BASE_TAG) \
+		--build-arg=PACKER_BUILD_NAME=$(subst _podman,,$*) \
```

Comment:
Re: Scripts using conditionals based on presence/absence of 'prior' appearing in the VM/Container image name.

> Can you get it from $VM_IMAGE_NAME?

That var. doesn't exist here (completely different Cirrus-CI setup).  In any case, it's value is derived from `$PACKER_BUILD_NAME` when producing VM images.  The problem area is with the container build workflow, as it shares the same setup/packaging scripts as the VM setup but does ***not*** use the packer tool.

Ordinarily the scripts could ignore this, as all conditions on `$PACKER_BUILD_NAME` only applied to VM's (i.e. containers don't care about setting kernel options)...except now this isn't the case, since I've un-commented the (previously commented in this PR) `$ENABLE_UPDATES_TESTING` conditional (on 'prior' flavors) for fedora.

I don't like this fix, it's convoluted and non-obvious :astonished: (esp. in the container image build case).  However, right now I cannot think of a better way to solve this in a more uniform yet readable manner, and am loathe to add a giant comment to the Makefile (which s already very dense and complex) :disappointed: 

@edsantiago  I'd really like to get this PR merged, since the previously built images were already rolled out all across container-org. CI-land.  Is this something we should just live with, fix later, ignore, or maybe I could file an issue on it (here) for fixing next year?

Reply:
Ooof...and it doesn't appear to work: [F32 container build log](https://cirrus-ci.com/task/6207579749613568?command=main#L598) *sigh*

Reply:
@edsantiago nvm my question, this is broken anyway so I'll clean it up and comment the heck out of it.